### DISCUSSION TITLE
Customizing the user menu - How to customize toggle dark mode link

### DISCUSSION THREAD

**Discussion by robyhartonodev** [filamentphp/filament#2966] [Help] [answered]
Hello. I'm trying to localize the user menu item label for toggling the dark mode. But I could not find the key for toggle dark mode user menu item in the documentation.

**What I mean:** 
![image](https://user-images.githubusercontent.com/17837057/176407434-d2526907-282e-432f-93bb-695d922f192f.png)

What I have tried so far inside the `AppServiceProvider` inside the `boot` method. But it doesn't work. According to the docs https://filamentphp.com/docs/2.x/admin/navigation#customizing-the-user-menu

```php
Filament::serving(function() {
            // ... other settings

            // Top right dropdown menu items
            Filament::registerUserMenuItems([
                'dark_mode'  => UserMenuItem::make()->label('Toggle Dark Mode')->url(route('logout')),
                'account'    => UserMenuItem::make()->url(route('filament.resources.users.profile', ['record' => $user->id])),
                'logout'     => UserMenuItem::make()->label('Log out')->url(route('logout')),
            ]);
}
```

**Comment by ryangjchandler** [comment, marked as answer]
The translation string can be found here: https://github.com/filamentphp/filament/blob/2c9255d29a755a6243035c77f7fcbd7d279fbc5e/packages/admin/resources/lang/en/layout.php

You just need to change this for your locale.

**Reply by robyhartonodev** [reply]
Thanks

**Reply by MarJose123** [reply]
How did you resolve? can you give me the full solution?

**Reply by robyhartonodev** [reply]
> How did you resolve? can you give me the full solution?

https://filamentphp.com/docs/2.x/admin/installation#publishing-the-translations
Publish the translation file `php artisan vendor:publish --tag=filament-translations` and just edit the file.

**Reply by MarJose123** [reply]
nothing happen when I click the toggle dark mode.
![image](https://user-images.githubusercontent.com/18107626/184293319-314b3af1-134f-462d-b015-771e409dc053.png)

here is my code under Service Provider
```php
class DarkModeServiceProvider extends ServiceProvider
{
    public function register()
    {

    }

    public function boot()
    {
        Filament::serving(function () {
            Filament::registerUserMenuItems([
                'dark_mode' => UserMenuItem::make()
                                ->label('Toggle dark mode')
                                ->url('#'),
                // ...
            ]);
        });
    }
}
```
and this service provider is also added on my `app.php` file can you guide me on how to make it work.

**Reply by danharrin** [reply]
Have you enabled dark mode in your config file?

**Reply by MarJose123** [reply]
> Have you enabled dark mode in your config file?

Thanks.
